ashpd-demo/src/portals/desktop/secret.rs

```diff
@@ -1,8 +1,4 @@
-use std::{
-    io::Read,
```

Comment:
this change belong into the clippy warnings commit

Reply:
it's produced by the build without clippy.

Reply:
done